finance/mip_raw_elt_batch.py

```diff
@@ -0,0 +1,285 @@
+import json
+import logging
+import os
+
+import backoff
+import pandas as pd
+import requests
+from requests.exceptions import RequestException
+import snowflake.connector as sc
+from snowflake.connector.pandas_tools import write_pandas
+import synapseclient
+from urllib3.exceptions import RequestError
+
+SECRETS = json.loads(os.getenv("SCHEDULED_JOB_SECRETS"))
+SYNAPSE_AUTH_TOKEN = SECRETS["SYNAPSE_AUTH_TOKEN"]
+SNOWFLAKE_PRIVATE_KEY = SECRETS["SNOWFLAKE_PRIVATE_KEY"]
+MIP_AUTH = SECRETS["MIP_AUTH"]
+
+with open("temp.p8", "w") as private_key_f:
+    private_key_f.write(SNOWFLAKE_PRIVATE_KEY)
+
+conn_params = {
+    "account": "mqzfhld-vp00034",
+    "role": "FINANCE_ADMIN",
+    "user": "FINANCE_SERVICE",
+    "private_key_file": "temp.p8",
+    "private_key_file_pwd": None,
+    "warehouse": "COMPUTE_XSMALL",
+    "database": "FINANCE",
+    "schema": "MIP_RAW",
+}
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
+
+_mips_url_login = "https://login.mip.com/api/v1/sso/mipadv/login"
+# https://documentation.mip.com/#/swagger/recursiveTransaction/transactions/recursiveTransactions
+_ledger_api = "https://api.mip.com/api/v1/recursiveTransactions/GeneralLedger/posted"
+# https://documentation.mip.com/#/swagger/maintenance/generalLedger/chartOfAccounts
+_chart_of_accounts_api = "https://api.mip.com/api/v1/maintain/ChartOfAccounts"
+_mips_url_logout = "https://api.mip.com/api/security/logout"
```

Comment:
team braphilianna: Why did you decide to put your [private variables](https://en.wikipedia.org/wiki/Naming_convention_(programming)#Python_and_Ruby) in the global namespace?